planning/behavior_velocity_planner/src/scene_module/intersection/util.cpp

```diff
@@ -131,7 +131,7 @@ bool splineInterpolate(
   if (output->points.size() > 1) {
     size_t l = resampled_s.size();
     output->points.front().point.pose.orientation = output->points.at(1).point.pose.orientation;
-    output->points.back().point.pose.orientation = output->points.at(l - 1).point.pose.orientation;
+    output->points.back().point.pose.orientation = output->points.at(l - 2).point.pose.orientation;
```

Comment:
output->points.at(l - 1) is where orientation is empty(=last point) so changed to l - 2(=last point - 1)

Reply:
@takayuki5168  @tkimura4 
can you take a look at this ?

Reply:
```
size_t l = resampled_s.size();
```
resampled_s and output->points are the same size, but it seems better to use output->points for readability.
This code seems ok for temporary fix (since using `l - 2` seems not so good).

For the future work, it's better to calculate orientation for all the points including front and back ones.
FYI, I've implemented spline interpolation for yaw angles, and I'll create a PR soon. (You don't have to use that function since I'm not sure why my PR will be merged)

Reply:
@takayuki5168 
fixed at 145aeb8